lib/babe/verify_test.go

```diff
@@ -911,8 +910,6 @@ func Test_verifyBlockEquivocation(t *testing.T) {
 		tt := tt
 
 		t.Run(tname, func(t *testing.T) {
-			//t.Parallel()
```

Comment:
Why?

Reply:
You could uncomment them. Does it cause race condition?

Reply:
Okay! Done